go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/cosign
 
-go 1.18
+go 1.19
```

Comment:
See my comment in rekor - can we skip just this one change?

Reply:
done